### ISSUE TITLE
How are error classes applied?

### ISSUE THREAD

**Issue by TheRouce** [derfenix/webarchive#5]
Hey, first wanna say I love the project, it's stupid fast and just as simple as it needs to be.

I'm working on a TailwindCSS theme for webarchive and I'm wondering how the error states work for the check icons on the main page. I can see where the classes are but how would I apply a class to a different div. I'm still early days with javascript so I apologize if this is a pretty simple question lol

Basically I just want to apply a different border color for each page along with having the check next to the title.

**Comment by derfenix** [comment]
Hi. Thanks for your feedback. The project is still in alpha, but I'm happy to know that it is useful for somebody.

I wish I'm undesrstand your question right. There is the code, that applies the error class - https://github.com/derfenix/webarchive/blob/3147a0b683215e4bc50cf572db043bb337b361ea/ui/basic/main.js#L17 So I'm just applying the class name equal to the page's status text representation. The css for this classes is here https://github.com/derfenix/webarchive/blob/3147a0b683215e4bc50cf572db043bb337b361ea/ui/basic/style.css#L50-L61
The status value here can be one of 

- **new** — url added, but processing still not started (should never met this status actually)
- **processing** — url processing is in progress
- **done** — processing finished with all formats success
- **failed** — processing was failed totally
- **with_errors** — some of formats was failed

**Comment by TheRouce** [comment]
Don't know how I missed that lol, but that's smart, thanks for the quick response
